crux.el

```diff
@@ -606,17 +606,25 @@ as the current user."
   (insert (format-time-string "%c" (current-time))))
 
 ;;;###autoload
-(defun crux-recentf-find-file ()
-  "Find a recent file using `completing-read'."
+(defun crux-recentf-find-file (&optional filter)
+  "Find a recent file using `completing-read'. If FILTER is
```

Comment:
The second sentence should start on its own line (otherwise the doc linter will complain). Mention that `FILTER` is supposed to be a function.